### PR TITLE
SIP-385 docs: updated readMe

### PR DESCRIPTION
**Pull request by old**

### CODE REVIEW

**Comment by iBotPeaches** [comment]
Looks like the linter isn't happy @old

**Comment by old** [comment]
> Looks like the linter isn't happy @old

fixed

File: README.md [hunk 1]
```diff
@@ -31,81 +31,255 @@ yarn add react-native-fs2
...
 
-#### stopDownload
 
-#### resumeDownload
+### `stopDownload`
+```ts
+// stopDownload(jobId: number): void
+await RNFS.stopDownload('JobID'): void
+```
+* Abort the current download job with this ID. The partial file will remain on the filesystem.
```
Comment (ian-wd) [inline README.md:110]: should we position this along with `resumeDownload` and `isResumable` next to the `downloadFile` documentation? that way users reading the `downloadFile` documentation will able to see all download related methods available in one group

File: README.md [hunk 2]
```diff
@@ -31,81 +31,255 @@ yarn add react-native-fs2
...
+    length: number = 0,
+    position: number = 0,
+    encoding?: Encoding
+  ): Promise<string> 
+*/
+
+const fileData = await RNFS.read('FileToRead', 0, 0, 'utf8')
+```
+* Reads bytes length from the given position of the file at `filepath` and returns the file `contents`. 
+* Optionally includes `EncodingOrOptions` with values:
```
Comment (iBotPeaches) [inline README.md:177]: This doc note mentions `EncodingOrOptions`, but example above is just `Encoding` type.

File: README.md [hunk 3]
```diff
@@ -31,81 +31,255 @@ yarn add react-native-fs2
...
+* Reads the `filepath` and returns its checksum as determined by algorithm, which can be one of the following `md5`  |  `sha1` | `sha224` | `sha256` | `sha384` | `sha512`.
 
-#### stat
+### `writeFile`
+```ts
+// writeFile(filepath: string, contents: string, encoding?: Encoding): Promise<void>
+await RNFS.write('FileToWrite', 'ContentsToWrite', 'utf8')
+```
+* Write the `contents` to `filepath`
+* Optionally includes `EncodingOrOptions` with values:
```
Comment (iBotPeaches) [inline README.md:195]: Same here, is `EncodingOrOptions` right type? - its also below on appendFile

**Review by iBotPeaches — APPROVED** [review]

**Review by jyann — APPROVED** [review]

**Pull request merged**